features/step_definitions/all_my_step_definitions_step.rb

```diff
@@ -0,0 +1,25 @@
+Given("the following user exists") do |table|
+    table.hashes.each do |user|
+      User.create(user)
+end
+  end
+  
+  Given("I am on the landing page") do
+    visit '/'
+  end
+  
+  When("I click on the link {string}") do |login|
```

Comment:
Please abstract this. This step definition will be used in other context than the login functionality and the name of the local variable should be more generic.